CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UNRELEASED] - 2024-??-??
 
+### Fixes
+- Fixed bug in `EdgeRankPlot` due to renaming of `edges` to `molecules` in `CellGraphAssay` objects
```

Comment:
I don't think this should be listed as a bug since it's actually an update of the metric name. Instead, I think we should write that we have made an update that makes `EdgeRankPlot` compatible with data sets processed with Pixelator Python > x.x.x. Just to be clear where the issue stems from.

Reply:
Ok, that's a good point